plugin/src/main/java/dev/regadas/trino/pubsub/listener/SchemaHelpers.java

```diff
@@ -386,7 +389,7 @@ private static Code toSchemaWarningCode(WarningCode warningCode) {
     }
 
     @SuppressWarnings("deprecation")
-    static Schema.SplitCompletedEvent from(SplitCompletedEvent event) {
+    static Schema.QueryEvent from(SplitCompletedEvent event) {
```

Comment:
Keep this return type and make a new `from(Schema.SplitCompledEvent event)`

Reply:
I did that first but not build for functions of same arguments different return type. A different name?

`Schema.QueryEvent toQueryEvent(SplitCompletedEvent event)`?

Reply:
sounds good.